Corona-Warn-App/src/main/res/values/legal_strings.xml

```diff
@@ -111,4 +111,22 @@
  <!-- YTXT: Second body section for Bugreporting share log Privacy Card -->
  <string name="debugging_debuglog_share_privacy_card_body_2" translatable="false">"If, after sending the report, you have provided your error report ID to the technical support team, then you may also request at any time that the other information you provided, the error report ID, and the error report itself be erased. Otherwise, the error report will be automatically erased after 14 days."</string>
 
+ <!-- Trace Location QR Info Privacy Strings -->
+ <!-- XHED: Title for privacy card -->
+ <string name="trace_location_privacy_card_title_text" translatable="false">"Data protection and data security"</string>
+ <!-- XTXT: Subtitle for privacy card -->
+ <string name="trace_location_privacy_card_subtitle_text" translatable="false">"If you tap on “Next”, you can create a QR code for your guests."</string>
+ <!-- XTXT: First bulletpoint title for privacy card -->
+ <string name="trace_location_privacy_card_first_bulletpoint_title_text" translatable="false">"Your event or place details will be stored in encoded form in a printable QR code."</string>
+ <!-- XTXT: First bulletpoint subtitle for privacy card -->
+ <string name="trace_location_privacy_card_fist_bulletpoint_subtitle_text" translatable="false">"Your guests can then scan the QR code with the app."</string>
+ <!-- XTXT: Second bulleptoint title for privacy card -->
+ <string name="trace_location_privacy_card_second_bulletpoint_title_text" translatable="false">"Please ensure that you are as specific as possible about the name, category and time of your event or place, and the typical length of stay."</string>
+ <!-- XTXT: Second vulletpoint subtitle for privacy card -->
+ <string name="trace_location_pricacy_card_second_bulletpoint_subtitle_text" translatable="false">"The more accurate your information, the more reliably your guests can be warned."</string>
+ <!-- XTXT: Third bulletpoint text for privacy card -->
+ <string name="trace_location_privacy_card_third_bulletpoint_text" translatable="false">"Please display the QR code in such a way that all your guests can scan it while observing the social distancing and hygiene rules (at least 1.5 metres)."</string>
```

Comment:
Hi @ralfgehrer could you please correct to 'meters' -> the lawyers have used British English here (metres), but the project uses American English (meters). Thanks.